File: 03-Booleans And Conditions/Conditions.java
public class Conditions {
    public static void main(String[] args) throws Exception {
        int x = 7;
        int y = 7;

        if(x >= y) {
            System.out.println("this is true");
        } else {
            System.out.println("this is false");
        }

        if(x > y) {
            System.out.println("x is greater than y");
        } else {
            if(x < y) {
                System.out.println("x is smaller than y");
            } else {
                System.out.println("x is equal to y");
            }
        }

        if(x > y){
            System.out.println("x is greater than y");
        } else if(x == y) {
            System.out.println("x is equal to y");
        } else {
            System.out.println("x is smaller than y");
        }
    }
}

File: 03-Booleans And Conditions/conditions.js
const x = 8;
const y = 7;

if(x >= y || x == 5) {
    console.log("this is true");
} else {
    console.log("this is false");
}

if(x > y){
    console.log("x is bigger");
} else {
    if(x == y) {
        console.log("x is equal to y");
    } else {
        console.log("x is smaller than y")
    }
}

if(x > y) {
    console.log("x is bigger");
} else if(x < y) {
    console.log("x is smaller than y")
} else {
    console.log("x is equal to y");
}